daffodil-test-ibm1/src/test/resources/test-suite/ibm-contributed/dpaext2.tdml

```diff
@@ -145,7 +145,7 @@
 
     <tdml:parserTestCase name="simple_type_properties_text_number_13_03" root="base_group"
 		model="./fvt/ext/dpa/dpanum_properties.dfdl.xsd" 
-		 description="Section 13.5 Specification of number base - 2,8, and 16">
+		 description="Section 13.5 Specification of number base - 2,8, and 16" roundTrip="twoPass">
```

Comment:
I would prefer to avoid having twoPass test cases, as they obscure what is actually going on. Can we instead update the document/infoset to match what we are expecting in the second pass. If we car about the details of the intermediate parses, it would be clearer to break this test into multiple tests that should the individual parse/unparse steps with roundTrip="None"

Reply:
I usually prefer two-pass over none since they both require that the parse matches the infoset, but you also get some unparse testing to at least make sure things unparse somewhat correctly. Which is all I'm really caring about in this particular test. I generally avoid changing IBM tests (unless they're completely broken), so this gives me some assurance that this is doing the right thing. (Note that two pass is required because there's a leading zero, which doesn't unparse).

The new tests added for textStandardBase are more strict, and do have separate tests for parse vs unparse when we want to ensure that unparse is doing the right thing. So while this test could potentially hide issues due to two pass, other tests should catch those issues.

Reply:
I'll also add that a few of other tests in this file use roundTrip="twoPass", so it is at least consistent with those.